pipeline/transformer/transformer_sierra/src/main/scala/uk/ac/wellcome/platform/transformer/sierra/transformers/MarcUtils.scala

```diff
@@ -32,4 +32,19 @@ trait MarcUtils {
       marcTag = marcTag,
       marcSubfieldTags = List(marcSubfieldTag)
     )
+
+  def getSubfieldContents(varField: VarField,
+                          tag: Option[String] = None): List[String] =
+    varField.subfields
+      .filter { subfield =>
+        tag match {
+          case Some(tag) => subfield.tag == tag
+          case None      => true
```

Comment:
Why do we pass true if None?

Reply:
Passing a `tag` argument to this function is optional, if we receive `None` then we assume they mean all contents regardless of tag

Reply:
(as required [here](https://github.com/wellcometrust/catalogue/blob/ca9f822e56ecaf72441560de8b1127ae42469547/pipeline/transformer/transformer_sierra/src/main/scala/uk/ac/wellcome/platform/transformer/sierra/transformers/SierraProduction.scala#L251))